engine/src/flutter/shell/platform/android/platform_view_android_delegate/platform_view_android_delegate.cc

```diff
@@ -37,6 +38,18 @@ void putStringAttributesIntoBuffer(
   }
 }
 
+void putStringIntoBuffer(const std::string& string,
+                         int32_t* buffer_int32,
+                         size_t& position,
+                         std::vector<std::string>& strings) {
+  if (string.empty()) {
+    buffer_int32[position++] = PlatformViewAndroidDelegate::kEmptyStringIndex;
+  } else {
+    buffer_int32[position++] = strings.size();
+    strings.push_back(string);
+  }
+}
+
```

Comment:
This function should be in an anonymous namespace.  You shouldn't have symbols with external linkage that don't have headers that point to them.  Same goes for these other functions in here

```c++
namespace flutter {
namespace {
void putStringIntoBuffer(const std::string& string,
                         int32_t* buffer_int32,
                         size_t& position,
                         std::vector<std::string>& strings) {...}
}
}
```